pages/linux/runuser.md

```diff
@@ -0,0 +1,23 @@
+# runuser
+
+> Run commands as a specific user and group without asking for password (needs root privileges).
```

Comment:
If it needs root privileges, perhaps we could prefix the commands themselves with `sudo` instead of mentioning it in the description?

This is what we've done in other pages.

Reply:
Hmm, this is intended to be literally run as root though, since it does not ask for password. Running it with sudo defeats the purpose of the command, for that you have `su`.

Reply:
Ah, I see! In that case this might be fine then.